features/step_definitions/affiliate_steps.rb

```diff
@@ -88,6 +88,11 @@
   site.update!(is_rss_govbox_enabled: true)
 end
 
+When /^the use redesigned results page is disabled for the site "(.*?)"$/ do |name|
```

Comment:
This seems unnecessary - `use_redesigned_results_page` defaults to false on affiliate creation, and can be easily/explicitly set in an affiliate creation given block like it's done here: https://github.com/GSA/search-gov/blob/391152e26c35d56b48d45c2ffe2fba42121aad99/features/admin_center_manage_display.feature#L657-L658.